version/derive/src/lib.rs

```diff
@@ -173,7 +173,7 @@ fn impl_version(args: Vec<NestedMeta>, item: TokenStream) -> TokenStream2 {
     let alias_type_name = format_ident!("_{}V{}", versioned_struct_name, version_number);
     quote!(
         /// Autogenerated alias type to link versioned item to its container.
-        type #alias_type_name = #struct_name;
+        pub type #alias_type_name = #struct_name;
```

Comment:
Why do we need this?

Reply:
good catch. This should remain private

Reply:
Yes, it probably should be `pub(super)`

Reply:
`pub(crate)` is also fine

Reply:
If in doubt, prefer `pub(crate)`. It makes re-arranging modules less annoying.